src/backend/distributed/shared_library_init.c

```diff
@@ -930,7 +930,22 @@ RegisterCitusConfigVariables(void)
 					 "performance benefits."),
 		&MultiShardCommitProtocol,
 		COMMIT_PROTOCOL_2PC,
-		multi_shard_commit_protocol_options,
+		shard_commit_protocol_options,
+		PGC_USERSET,
+		0,
+		NULL, NULL, NULL);
+
+	DefineCustomEnumVariable(
+		"citus.single_shard_commit_protocol",
```

Comment:
Minor: I was thinking to make it `GUC_NO_SHOW_ALL ` since this is a GUC we expect to be used very rare (and hopefully never).

If you agree with me, please mark the GUC with `GUC_NO_SHOW_ALL`.